.travis.yml

```diff
@@ -15,16 +19,16 @@ virtualenv:
 
 before_install:
     - sudo add-apt-repository -y ppa:reddit/ppa
+    - echo "deb http://debian.datastax.com/community stable main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+    - curl -L https://debian.datastax.com/debian/repo_key | sudo apt-key add -
     - sudo apt-get update -q
-    - sudo apt-get install -y python-fbthrift python3-fbthrift fbthrift-compiler python-sphinx python-sphinxcontrib.spelling python-alabaster python-posix-ipc python3-posix-ipc python-enum34 python-flake8 pylint python-redis python-gevent python-pymemcache python3-pymemcache
-
-install:
-    - pip install -e ".[pyramid]"
-    - pip install webtest coverage
+    - sudo apt-get install -y cassandra=2.2.8 python python3 fbthrift-compiler make pylint python-flake8 python3-cassandra python3-coverage python3-cqlmapper python3-fbthrift python3-gevent python3-nose python3-posix-ipc python3-pymemcache python3-redis python3-requests python3-setuptools python3-sqlalchemy python3-webtest python-coverage python-cqlmapper python-fbthrift python-gevent python-nose python-posix-ipc python-pymemcache python-pyramid python-redis python-requests python-setuptools python-sqlalchemy python-webtest python-alabaster python-sphinx python-sphinxcontrib.spelling python-enum34 python-mock
```

Comment:
🔕  I know in you can break this one line apart like

```
    - sudo apt-get install -y cassandra=2.2.8 
        python python3 
        fbthrift-compiler 
        make 
        pylint 
        python-flake8 
        ...
```

If that works with Travis it might be worth doing to make it easier to keep track of updates to this.